app.ipynb.example

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Should this file be in a directory instead of top-level? We might have a need to have multiple notebooks down the road so carving out that space right now would probably be good.

Reply:
Agreed. I'm planning on adding a directory for creating seed data as well. Wondering if there should be a single top level directory that contains all of these notebooks/scripts/etc. that are only relevant to local development.